### PR TITLE
[4.0] Remove [].slice.call

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue # .

### Summary of Changes

As stated in https://volunteers.joomla.org/departments/production/reports/1031-production-dept-meeting-may-28-2019, J4 will no longer support IE11, therefore we do not require `[].slice.call(...)`

### CODE REVIEW

File: build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ tinymce.PluginManager.add('jdragdrop', (editor) => {
     editor.on('drop', (e) => {
       // We override only for files
       if (e.dataTransfer && e.dataTransfer.files && e.dataTransfer.files.length > 0) {
-        const files = [].slice.call(e.dataTransfer.files);
+        const files = e.dataTransfer.files;
```
[the remark this answers is not in the document]
Comment (dgrammatiko) [reply build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js]: `const { files } = e.dataTransfer;`

**Comment by dgrammatiko** [comment]
@C-Lodder are you sure that all evergreen browsers treat nodelists as arrays? if not you can convert the `[].slice()` to `Array.from()`

File: build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (C-Lodder) [reply build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js]: yup dw, sorted

**Comment by C-Lodder** [comment]
@dgrammatiko pretty sure

**Comment by dgrammatiko** [comment]
According to MDN https://developer.mozilla.org/en-US/docs/Web/API/NodeList
Opera for Android and Edge Mobile don't support `nodeList.forEach()`

**Comment by C-Lodder** [comment]
@dgrammatiko It says the support it unknown. I'll give this a test tomorrow on my phone

**Comment by C-Lodder** [comment]
@dgrammatiko - Tested on Edge and Opera for mobile and looping through a nodeList with `forEach` works fine:

https://jsfiddle.net/sjn8z0e7/1/

I'm going to submit a PR to MDN to rectify this.

**Comment by C-Lodder** [comment]
PR submitted: https://github.com/mdn/browser-compat-data/pull/4288

and Hound issues fixed

**Pull request closed without merging**